contracts/test/hardhat/contracts/FleekERC721/get-token.t.ts

```diff
@@ -0,0 +1,28 @@
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { expect } from 'chai';
+import { TestConstants, Fixtures, parseTokenURI } from './helpers';
+import { ethers } from 'hardhat';
+const { MintParams } = TestConstants;
+
+describe('FleekERC721.GetToken', () => {
+  let fixture: Awaited<ReturnType<typeof Fixtures.withMint>>;
+
+  before(async () => {
+    fixture = await loadFixture(Fixtures.withMint);
```

Comment:
maybe here would be better be `beforeEach` since if we add more tests, if one of them change the fixture the change will be kept to other tests.